charts/kubedb-grafana-dashboards/dashboards/kafka/kafka-2.json

```diff
@@ -0,0 +1,3684 @@
+{{- $shared := and (eq .Values.app.name "") (eq .Values.app.namespace "") -}}
+{{- $alerts := (eq $.Values.dashboard.alerts true) -}}
+{
```

Comment:
What is this `kafka-2` file ? Mistakenly pushed ?

Reply:
Yes. deleted